<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>220</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Battery Status</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="timeoutComboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="connectTypeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>До полной разрядки</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeToLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>520</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeoutLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>150</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Тайм-аут экрана:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="levelProgressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>531</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
